docs/Makefile

```diff
@@ -5,11 +5,12 @@
 SPHINXOPTS    = -W
 SPHINXBUILD   = sphinx-build
 PAPER         =
+BUILDDIR      = _build
```

Comment:
I matched the Makefile with the one from `pyramind_tm` - let me know if you want me to revert this change.

Reply:
I think that this is just an older `Makefile` vs the one that is used in `pyramid_tm` due to `pyramid_tm` being a newer project (IIRC).